acms-react-app/src/Services/customer.service.js

```diff
@@ -15,6 +15,22 @@ class customerService {
         return axios.get(`${API_AVAILABILITY_URL}${date}`, { headers: authHeader() });
     }
 
+    // reviews (good practice but maybe redundant in this case?)
+
```

Comment:
we should've discussed this before the implementation, but everything that has to do with the user side of things go here and everything that has to do with the admin side of things to in the admin.service. You implemented everything in the admin's side which will still work but it will be very confusing to understand when reading the code.